### PR TITLE
WIP Email Validation

### PR DESCRIPTION
**Pull request by niamtokik**

### CODE REVIEW

File: lib/sna_web/controllers/auth_controller.ex [hunk 1]
```diff
@@ -8,6 +8,7 @@ defmodule SnaWeb.AuthController do
   def email(conn, %{"email" => email}) do
     case SnaWeb.Token.generate_bearer(%{"email" => email}) do
       {:ok, token, _claims} ->
+        SnaWeb.Mail.token_validation(conn, email, token)
         render(conn, "email.html", email: email, token: token)
```
Comment (mildred) [inline lib/sna_web/controllers/auth_controller.ex:12]:
```suggestion
        render(conn, "email.html", email: email)
```

File: lib/sna_web/mail.ex [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  @doc """
+  
+  deliver_now/1 is a wrapper around Sna.Mail.deliver_now/1 and permit
+  to send an email.
+  
+  """
+  @spec deliver_now(Bamboo.Email.t()) :: Bamboo.Email.t()
+  def deliver_now(mail) do
+    mail |> Sna.Mailer.deliver_now
+  end
```
Comment (mildred) [inline lib/sna_web/mail.ex:51]: Why wrap Sna.Mailer.deliver_now and not use it directly?

**Review by mildred — APPROVED** [review]

**Comment by kwa29** [comment]
Need to rebase this PR?

**Comment by niamtokik** [comment]
probably archive SNA soon. New projects are coming.

**Pull request closed without merging**